src/cplscheme/BaseCouplingScheme.cpp

```diff
@@ -676,34 +676,34 @@ void BaseCouplingScheme::initializeTXTWriters()
       if (convMeasure.level > 0)
         hasCoarseModelOptimization = true;
 
-    _iterationsWriter->addData("Timesteps", io::TXTTableWriter::INT);
-    _iterationsWriter->addData("Total_Iterations", io::TXTTableWriter::INT);
+    _iterationsWriter->addData("Timestep", io::TXTTableWriter::INT);
+    _iterationsWriter->addData("TotalIterations", io::TXTTableWriter::INT);
     _iterationsWriter->addData("Iterations", io::TXTTableWriter::INT);
     if (hasCoarseModelOptimization) {
-      _iterationsWriter->addData("Total_Iterations_Surrogate_Model", io::TXTTableWriter::INT);
-      _iterationsWriter->addData("Iterations_Surrogate_Model", io::TXTTableWriter::INT);
+      _iterationsWriter->addData("TotalIterations_SurrogateModel", io::TXTTableWriter::INT);
```

Comment:
I understand you are trying to nicely separate between `TotalIterations` and `TotalIterations_SurrogateModel`, but wouldn't just `TotalIterationsSurrogateModel` be good (and consistent) as well? Does [clang-format](http://xgm.de/precice/docs/develop/tooling.html) suggest any change?